controllers/suite_test.go

```diff
@@ -65,6 +65,9 @@ var _ = BeforeSuite(func(done Done) {
 	err = mongodbcomv1.AddToScheme(scheme.Scheme)
 	Expect(err).NotTo(HaveOccurred())
 
+	err = mongodbcomv1.AddToScheme(scheme.Scheme)
```

Comment:
Is this generated as well? Doesn't look right

Reply:
yes, you are right, will remove 👍